json/Card/hisu-mori-toride-elements-unbound.json

```diff
@@ -0,0 +1,29 @@
+[
+	{
+		"clan": "unicorn",
+		"cost": null,
+		"deck_limit": 1,
+		"element": null,
+		"fate": 7,
+		"glory": null,
+		"honor": 11,
+		"id": "hisu-mori-toride-elements-unbound",
```

Comment:
I'm not in favour of tying a card to a pack. Anyway, it can't work right now because the id has to be deduced from the name alone. I need to change that, and I'm still not sure how. I was thinking adding a "note" attribute, then use name+note to get the id.